XIVSlothCombo/Combos/PvE/SAM.cs

```diff
@@ -537,7 +537,7 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                                         if (IsEnabled(CustomComboPreset.SAM_ST_GekkoCombo_CDs_OgiNamikiri_Burst))
                                         {
-                                            if (hasDied || nonOpener || (meikyostacks is 1 or 2 && GetDebuffRemainingTime(Debuffs.Higanbana) >= 45 && HasEffect(Buffs.MeikyoShisui)) || GetCooldownRemainingTime(Ikishoten) <= 105)
+                                            if (hasDied || nonOpener || (meikyostacks is 1 or 2 && GetDebuffRemainingTime(Debuffs.Higanbana) >= 45 && HasEffect(Buffs.MeikyoShisui)) || GetCooldownRemainingTime(Ikishoten) <= 105 || enemyHP < executeThreshold)
```

Comment:
Would this need extra checks outside of "hp% < threshold" to prevent `Ogi Namikiri` from blocking the combo up?